README.md

```diff
@@ -1 +1,62 @@
-# client-examples
\ No newline at end of file
+# Description of client-examples
+This repository contains example of Kafka clients. 
+Both producer and consumer clients are transformed into docker images which allows to deploy them on OpenShift platform.
+This may serve as basic usage example for [Strimzi][strimziGH]project.
+
+This repository also contains `deployment.yaml` file with full deployment configuration of producer, consumer, and definitions for topic and user operator.
+Logging configuration can be found in the `log4j2.properties` file for producer and consumer separately.
+
+## Build
+After cloning this repository to your folder Hello World example is fully ready to be build with maven.
+By one single command Java sources are compiled into JAR files, docker images are created and pushed to repository.
+The command is:
+    `make all`
+
+Note: Be sure `docker` and `oc` cluster where images should be pushed are running.
+
+## Usage
+This example provides Hello World use case of Strimzi Kafka Operator.
+Basic requirement to run this example is running OpenShift cluster with deployed Kafka and Zookeeper containers.
+Examples how to deploy basic configuration can be found in the [Strimzi documentation][strimziDoc].
+
+After successful build and push images into repository you are ready to deploy producer and consumer containers along with Kafka and Zookeper.
+This can be done by applying `helloworld-producer.yaml` and `helloworld-consumer.yaml` files.
+This deploys producer and consumer images ready to publish and subscribe to defined topic.
+Second option is to apply `deployment.yaml` file. This deploys producer and consumer and also creates topic and example is in ready-to-observe state.
```

Comment:
"The second option" (but if there are two options you should say at the start, or use a bullet list)

"This deploys the producer and consumer ans also creates the topic..."